src/app/containers/MostRead/Canonical/index.jsx

```diff
@@ -52,14 +65,39 @@ const CanonicalMostRead = ({ endpoint }) => {
   }, [endpoint, numberOfItems, datetimeLocale, lastUpdated, script, service]);
 
   return items.length ? (
-    <MostRead
-      items={items}
-      header={header}
-      service={service}
-      script={script}
-      dir={dir}
-      labelId="Most-Read"
-    />
+    // eslint-disable-next-line jsx-a11y/no-redundant-roles
+    <section role="region" aria-labelledby="Most-Read">
+      <SectionLabel
+        script={script}
+        labelId="Most-Read"
+        service={service}
+        dir={dir}
+      >
+        {header}
+      </SectionLabel>
+      <MostReadList numberOfItems={items.length} dir={dir}>
```

Comment:
I believe we should be adding the same spacing between the `Section Label` and the `MostReadList`, to keep it consistent with the rest of sections https://github.com/bbc/simorgh/blob/latest/src/app/containers/FrontPageSection/index.jsx#L115

Reply:
good spot, will add this now 🙇